servers/src/test/java/tachyon/worker/block/TieredBlockStoreTests.java

```diff
@@ -424,18 +423,16 @@ public void moveTempBlockTest() throws Exception {
     mBlockStore.moveBlock(SESSION_ID1, TEMP_BLOCK_ID, mTestDir2.toBlockStoreLocation());
   }
 
-  // TACHYON-825
-  @Ignore
   @Test
-  public void moveBlockToTheLocationWithExistingIdTest() throws Exception {
+  public void cacheSameBlockInDifferentTiersTest() throws Exception {
```

Comment:
Is there a test case for caching the same block in different dirs of the same tier?